rust/src/client/mobile_client/client.rs

```diff
@@ -0,0 +1,364 @@
+use crate::{
+    client::{
+        mobile_client::participant::{
+            Participant,
+            ParticipantSettings,
+            Sum,
+            Sum2,
+            Type,
+            Undefined,
+            Update,
+        },
+        ClientError,
+        Proxy,
+    },
+    crypto::ByteObject,
+    mask::model::Model,
+    state_machine::coordinator::RoundParameters,
+    InitError,
+    PetError,
+};
+use derive_more::From;
+use std::{cell::RefCell, rc::Rc};
+
+#[derive(From)]
+pub enum ClientStateMachine {
+    Undefined(ClientState<Undefined>),
```

Comment:
`Undefined` seems a bit odd to me, it is actually well defined by those very lines of code^^. what do you think of `Pending` or `Dangling` or sth similar?